app-config-electron/README.md

```diff
@@ -0,0 +1,49 @@
+# App Config Electron
+
+Easily inject `app-config` values into Electron renderer processes. See `examples/electron-project` for an example project.
+
+## Usage
+
+### 1. Install `@app-config/main` and `@app-config/electron`
+
+```shell
+yarn add @app-config/main
+yarn add @app-config/electron
+```
+
+Or, if you use NPM.
+
+```shell
+npm i @app-config/main
+npm i @app-config/electron
+```
+
+### 2. Load your config with any options you need in the main Electron process before creating any windows
+```typescript
+app.whenReady().then(() => {
+  loadConfig().then(() => {
+    const mainWindow = new BrowserWindow();
+
+    mainWindow.loadFile('./index.html')
+  });
+});
+```
+
+### 3. Insert the App Config preload script into your `BrowserWindow` `webPreferences`
+
+Pass your config and optionally any other `BrowserWindow` `webPreferences` you need to `addAppConfigPreload`.
+```typescript
+app.whenReady().then(() => {
+  loadConfig().then(() => {
+    const mainWindow = new BrowserWindow({
+      webPreferences: addAppConfigPreload(config),
+    });
+
+    mainWindow.loadFile('./index.html')
+  });
+});
+```
+
+### 4. Your App Config values are now available under `window._appConfig` or in `config` if using `@app-config/main` in your web page. It's also available in `config` in the main Electron process.
```

Comment:
Could we make the titles less wordy? See [rendered](https://github.com/launchcodedev/app-config/blob/48e57e39f50e5542529bcdf199b0f5fdf9f19406/app-config-electron/README.md)